src/index.js

```diff
@@ -31,19 +31,39 @@ const closeArgsToError = (code, signal) => {
   return null
 }
 
+const concatBuffer = (buffer) => {
+  if (buffer.length) {
+    return
+  } else if (typeof buffer[0] === 'string') {
+    result = buffer.join('')
+  } else if (buffer[0] instanceof Buffer) {
+    result = Buffer.concat(buffer)
+  } else {
+    // throw new Error('Unexpected buffer type')
+    return
```

Comment:
This case probably shouldn't occur but I'm not sure whether to throw error or just return?